examples/plugins/ebpf/main.go

```diff
@@ -0,0 +1,179 @@
+package main
```

Comment:
Probably yes. and would be a good example. I don't really know enough python to write it well.